<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowhist">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a_max (0 ~ 1)</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtAlpha">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInvert">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma(1,2...)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdgeDetection">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Detection</string>
    </property>
   </widget>
   <widget class="QSlider" name="hsKValue">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblKvalue">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtSigma">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectWebcam">
    <property name="geometry">
     <rect>
      <x>704</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopWebcam">
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in blur</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtBeta">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur(1,2...)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1274</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
